docs/_data/bearer_scan.yaml

```
@@ -1,103 +1,107 @@
-    - name: no-color
-      default_value: "false"
-      usage: Disable color in output
-    - name: only-rule
-      default_value: '[]'
-      usage: |
-        Specify the comma-separated ids of the rules you would like to run. Skips all other rules.
-    - name: output
-      usage: Specify the output path for the report.
-    - name: parallel
-      default_value: "0"
-      usage: Specify the amount of parallelism to use during the scan
-    - name: quiet
-      default_value: "false"
-      usage: Suppress non-essential messages
-    - name: report
-      default_value: security
-      usage: Specify the type of report (security, privacy, dataflow).
-    - name: scanner
-      default_value: '[sast]'
-      usage: |
-        Specify which scanner to use e.g. --scanner=secrets, --scanner=secrets,sast
-    - name: severity
-      default_value: critical,high,medium,low,warning
-      usage: Specify which severities are included in the report.
-    - name: skip-path
-      default_value: '[]'
-      usage: |
-        Specify the comma separated files and directories to skip. Supports * syntax, e.g. --skip-path users/*.go,users/admin.sql
-    - name: skip-rule
-      default_value: '[]'
-      usage: |
-        Specify the comma-separated ids of the rules you would like to skip. Runs all other rules.
+  - name: api-key
+    usage: Use your Bearer API Key to send the report to Bearer.
+  - name: config-file
+    default_value: bearer.yml
+    usage: Load configuration from the specified path.
+  - name: context
+    usage: |
+      Expand context of schema classification e.g., --context=health, to include data types particular to health
+  - name: data-subject-mapping
+    usage: |
+      Override default data subject mapping by providing a path to a custom mapping JSON file
+  - name: debug
+    default_value: "false"
+    usage: Enable debug logs. Equivalent to --log-level=debug
+  - name: debug-profile
+    default_value: "false"
+    usage: Generate profiling data for debugging
+  - name: disable-default-rules
+    default_value: "false"
+    usage: Disables all default and built-in rules.
+  - name: disable-domain-resolution
+    default_value: "true"
+    usage: |
+      Do not attempt to resolve detected domains during classification
+  - name: disable-version-check
+    default_value: "false"
+    usage: Disable Bearer version checking
+  - name: domain-resolution-timeout
+    default_value: 3s
+    usage: |
+      Set timeout when attempting to resolve detected domains during classification, e.g. --domain-resolution-timeout=3s
+  - name: exclude-fingerprint
+    default_value: "[]"
+    usage: |
+      Specify the comma-separated fingerprints of the findings you would like to exclude from the report.
+  - name: exit-code
+    default_value: "-1"
+    usage: |
+      Use this to force a given exit code for the scan command. Set this to 0 (success) to always return a success exit code despite any findings from the scan.
```

Seeing the others, maybe this is more consistent. Sorry for the back and forth 🙈 
```suggestion
      Force a given exit code for the scan command. Set this to 0 (success) to always return a success exit code despite any findings from the scan.
```

---

No pb. Agreed!